app/controllers/api/v1/deals_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Api::V1
+  class DealsController < ApplicationController
+    def index
+      @deals = Deal.all
+      render json: @deals
+    end
+
+    def accounting
+      render json: {
+        deals: Deal.paginate(page: page)
+                   .order(sort_by + ' ' + order),
+        page: page,
+        pages: Deal.pages
+      }
+    end
+
+    def accounting_create_deal
```

Comment:
These are not restful actions.  I would say lets talk about this in person before changing any of the code.

Reply:
controllers don't have to be restful 100% of the time but you should understand the tradeoffs and options.  i know you have been watching dhh stuff.  http://jeromedalbert.com/how-dhh-organizes-his-rails-controllers/